projects/plugins/jetpack/extensions/blocks/instagram-gallery/instagram-gallery.php

```diff
@@ -61,8 +61,7 @@ function render_block( $attributes, $content ) { // phpcs:ignore VariableAnalysi
 		)
 	);
 
-	$grid_style  = 'grid-gap: ' . $spacing . 'px;';
-	$photo_style = 'padding: ' . $spacing . 'px;';
+	$grid_style = sprintf( 'grid-gap: %spx; --latest-instagram-posts-spacing: %spx;', $spacing, $spacing );
```

Comment:
Maybe this would simplify things a bit?

```suggestion
	$grid_style = sprintf(
		'grid-gap: %1$spx; --latest-instagram-posts-spacing: %1$spx;',
		$spacing
	);
```

Reply:
Thanks for that 👍